packages/eslint-plugin/docs/rules/no-unnecessary-type-parameters.mdx

```diff
@@ -76,16 +76,22 @@ function getProperty<T, K extends keyof T>(obj: T, key: K) {
 ## Limitations
 
 Note that this rule allows any type parameter that is used multiple times, even if those uses are via a type argument.
-For example, the following `T` is used multiple times by virtue of being in an `Array`, even though its name only appears once after declaration:
+For example, the following `T` is used multiple times by virtue of being in an `Set`, even though its name only appears once after declaration:
```

Comment:
Now that this PR doesn't touch _return_ location arrays, only _parameter_ location ones, these docs don't need to change anymore. I'll go ahead and revert the now-unnecessary changes.

```suggestion
For example, the following `T` is used multiple times by virtue of being in an `Array`, even though its name only appears once after declaration:
```